### PR TITLE
Rename _getSize() to _sizeFromOffer()

### LINKED ISSUE #2059: [Component] method renames: Layout
`requestedSpace()` makes sense, but the distinction between `computeLayout()` and `protected _getSize()` could be made clearer.

### PR DESCRIPTION
**Pull request by jtlan**
New name better explains the purpose of the method and differentiates it from `requestedSpace()` and `computeLayout()`.

Close #2059.

### CODE REVIEW

File: src/components/axes/timeAxis.ts [hunk 1]
```diff
@@ -304,11 +304,11 @@ export module Axes {
       return Math.min(this._getIntervalLength(config), this.width()) >= worstWidth;
     }
 
-    protected _getSize(availableWidth: number, availableHeight: number) {
+    protected finalSizeFromOffer(availableWidth: number, availableHeight: number) {
```
Comment (aicioara) [inline src/components/axes/timeAxis.ts]: Underscore, this is protected

File: src/components/axes/timeAxis.ts [hunk 2]
```diff
@@ -304,11 +304,11 @@ export module Axes {
...
       // Makes sure that the size it requires is a multiple of tier sizes, such that
       // we have no leftover tiers
 
-      var size = super._getSize(availableWidth, availableHeight);
+      var size = super.finalSizeFromOffer(availableWidth, availableHeight);
```
Comment (aicioara) [inline src/components/axes/timeAxis.ts]: I am confused. How does this compile?

File: src/components/axes/timeAxis.ts (hunk 1 above)
Comment (aicioara) [inline src/components/axes/timeAxis.ts]: Also, why not `_sizeFromOffer`?

Comment (jtlan) [inline src/components/axes/timeAxis.ts]: I thought I had it as `_sizeFromOffer(). Looks like I goofed.

**Pull request closed without merging**